src/apiCalls.js

```diff
@@ -1,4 +1,3 @@
 export const getOrders = () => {
-  return fetch('http://localhost:3001/api/v1/orders')
-      .then(response => response.json())
-}
\ No newline at end of file
+  return fetch("URL").then((response) => response.json());
```

Comment:
Can you tell me more about this change?  Why did we replace the endpoint url with a string "URL"?

Reply:
Thank you for asking this question @hfaerber! To be honest, I've noticed that students often don't read the documentation or ReadMe thoroughly, and as a result, they may miss out on important information. That's why I changed the URL to encourage them to check the URL from the ReadMe and try making the request themselves.

Reply:
I revert back to the Base URL